bedrock/firefox/templates/firefox/desktop/trust.html

```diff
@@ -48,25 +48,28 @@
       </p>
 
       <ul class="feature-list">
-        <li id="privacy-dnt">
-          <h3>{{ _('Do Not Track') }}</h3>
+        {% if l10n_has_tag('tracking-protection') %}
```

Comment:
Can you switch to `tracking_protection` like the other page? We should not have issues anymore with hyphens, but still, underscore are safe and consistent ;-)